crates/core/src/db/update.rs

```diff
@@ -221,6 +229,20 @@ fn auto_migrate_database(
             spacetimedb_schema::auto_migrate::AutoMigrateStep::RemoveSchedule(_) => {
                 anyhow::bail!("Removing schedules is not yet implemented");
             }
+            spacetimedb_schema::auto_migrate::AutoMigrateStep::AddRowLevelSecurity(sql_rls) => {
+                system_logger.info(&format!("Adding row-level security `{sql_rls}`"));
+                log::info!("Adding row-level security `{sql_rls}`");
+                let tables = all_tables.values().cloned().collect::<Vec<_>>();
+                let rls = RawRowLevelSecurityDefV9::lookup(plan.new, sql_rls).unwrap();
```

Comment:
Yes, this is sad. I ponder the possibility of adding an alternative `lookup`

Reply:
What about resolving the rls filters **after** all the other auto-migrate steps?

Reply:
Done